### ISSUE TITLE
Any Multi-language Plan? 

### ISSUE THREAD

**Issue by once375ml** [openscopeproject/TrguiNG#123]
If so, I would like to contribute the simplified Chinese translation.

Thanks.

**Comment by qu1ck** [comment]
Yes, I plan to add i18n at some point. I have to choose a service, something like transifex and a library that can work.

**Comment by JustAndreww** [comment]
Crowdin also a nice collaborative translation service.

**Comment by Melody-Test** [comment]
Expect to have Simplified Chinese

**Comment by dongpei151** [comment]
Expect to have Simplified Chinese

**Comment by qu1ck** [comment]
Just to give an update on this, I will most likely use Weblate service for translations and implement this when number of features added slows down to the point where I can easily do a string freeze for a few weeks before each release.
Right now new stuff is being added more or less regularly and I want to be able to push out a release any time without waiting on translators.

**Comment by yalidabuda** [comment]
Very much looking forward to it！

**Comment by starmoonxp** [comment]
+1 Simplified Chinese

**Comment by once375ml** [comment]
> Expect to have Simplified Chinese

https://github.com/jayzcoder/TrguiNG

Someone has done.

**Comment by once375ml** [comment]
> +1 Simplified Chinese

https://github.com/jayzcoder/TrguiNG

Someone has done.

**Comment by ManuZhu0728** [comment]
太长不看版：用AI编程憋了个中文汉化，移植了几个原汉化的功能，勉强能用吧😐指路https://github.com/ManuZhu0728/TrguiNG


Hi everyone,

Regarding the i18n support, I created a fork and implemented basic support for **Simplified & Traditional Chinese** to meet my own daily needs.

**✨ Features Added:**
* **i18n Support:**
    * **Simplified Chinese:** Initially generated by AI, then manually refined referencing [jayzcoder/TrguiNG](https://github.com/jayzcoder/TrguiNG).
    * **Traditional Chinese:** AI-converted from Simplified Chinese.
    * **Other Language:** Support for additional languages can be added by creating translation files in `src/locales/`.
* **Ported Features** (from [jayzcoder/TrguiNG](https://github.com/jayzcoder/TrguiNG)): Group size display, Double-click group header to select all, Dedicated "Error" grouping, Different layout mode.

**⚠️ Disclaimer:**
This implementation was done via **rapid prototyping (Vibe Coding)** to get it working quickly, utilizing GPT-5.1 for the framework and Gemini 3 Pro for translations.
Consequently, the code quality may not meet strict standards for a direct merge, and the translations might contain errors. **Please feel free to point out any inaccuracies.**

It works... sorta😐

* **For users:** You are welcome to try it if you need Chinese immediately.
* **For the maintainer:** First of all, **thank you for your hard work on this project!** I hope my fork can serve as a reference or a "proof of concept" when you decide to implement official support with Weblate later.

**My fork:** https://github.com/ManuZhu0728/TrguiNG
